packages/framework/ai-collab/package.json

```diff
@@ -61,8 +61,8 @@
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
 		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
-		"test:mocha:cjs": "echo \"TESTS ARE CURRENTLY FAILING. SKIPPING RUN.;\" # mocha --recursive \"dist/test/**/*.spec.js\"",
-		"test:mocha:esm": "echo \"TESTS ARE CURRENTLY FAILING. SKIPPING RUN.;\" # mocha --recursive \"lib/test/**/*.spec.js\"",
+		"test:mocha:cjs": "mocha --recursive \"dist/test/**/*.spec.js\"",
+		"test:mocha:esm": "mocha --recursive \"lib/test/**/*.spec.js\" --timeout 60000",
```

Comment:
I don't think we should add unit tests that reach out to external systems, especially if they can take up to a minute to run. That kind of test should go in the e2e tests package, or at the very least a different script in this package.

Reply:
the timeout was left over from integ testing. I've updated the script to remove the leftover timeout. I agree that these integ tests should be eventually moved to e2e when we have open ai credentials and whatnot ready to go in the pipeline.

This script is updated in commit bac309434408ff1fd2adfbc0dcacf19c89f40ee8